### PR TITLE
[GITHUB-1] Imports PHP bits from newrelic role

### PR DESCRIPTION
**Pull request by lobsterdore**
Copies in PHP integration bits from the existing newrelic role
with a few bits of cleanup along the way.

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Configure New Relic PHP agent
+  become: yes
+  ini_file:
+    dest: "{{ sansible_newrelic_agent_php_ini_paths_base }}"
+    mode: 0644
+    section: "{{ item.section | default(None) }}"
+    option: "{{ item.option }}"
+    value: "{{ item.value }}"
```
Comment (dennisconrad) [inline tasks/configure.yml]: Due to the ongoing Ansible boolean issues, may I recommend:

```yaml
    value: "{% if item.value == true %}true{% elif item.value == false %}false{% else %}{{ item.value }}{% endif %}"
```

**Review by dennisconrad — APPROVED** [review]

File: tasks/configure.yml (hunk 1 above)
Comment (lobsterdore) [reply tasks/configure.yml]: Good idea, done and done

File: tasks/configure.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Configure New Relic PHP agent
+  become: yes
+  ini_file:
+    dest: "{{ sansible_newrelic_agent_php_ini_paths_base }}"
+    mode: 0644
+    section: "{{ item.section | default(None) }}"
+    option: "{{ item.option }}"
+    value: "{% if item.value == true %}true{% elif item.value == false %}false{% else %}{{ item.value }}{% endif %}"
```
Comment (wojtek-oledzki) [inline tasks/configure.yml:10]:
```suggestion
    value: "{{ 'true' if item.value == true else 'false' }}" 
```

**Review by wojtek-oledzki — COMMENTED** [review]
small change suggestion

File: tasks/configure.yml (hunk 2 above)
Comment (lobsterdore) [reply tasks/configure.yml:10]: That removes the third case though @wojtek-oledzki ?

Comment (lobsterdore) [reply tasks/configure.yml:10]: This was a suggestion from @dennisconrad, he's noticed that Ansible can often convert string representations of booleans behind the scenes and produce unexpected results.

Comment (dennisconrad) [reply tasks/configure.yml:10]: That's not the same unfortunatly, @wojtek-oledzki

Comment (wojtek-oledzki) [reply tasks/configure.yml:10]: oh sorry - I've missed the elseif
```
{{ 'true' if item.value == true else 'false' if item.value == false else item.value }}
```

File: README.md [hunk 3]
```diff
@@ -27,18 +28,58 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
...
+Enable default PHP7.3 agent:
+
+```YAML
+- name: Install and configure newrelic
+  hosts: "somehost"
+
+  roles:
+    - role: sansible.newrelic_agent_php
+      sansible_newrelic_agent_php_appname: some-php7.3-app
+      sansible_newrelic_license_key: 123456789123456789123456789123456789
```
Comment (wojtek-oledzki) [inline README.md]:
```suggestion
      sansible_newrelic_agent_php_license_key: 123456789123456789123456789123456789
```

File: README.md [hunk 4]
```diff
@@ -27,18 +28,58 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
...
+  hosts: "somehost"
+
+  roles:
+    - role: sansible.newrelic
+      sansible_newrelic_agent_php_appname: some-php7.2-app
+      sansible_newrelic_agent_php_ini_paths_base:  /etc/php/7.2/mods-available/newrelic.ini
+      sansible_newrelic_agents_php_ini_paths_links:
+      - /etc/php/7.2/fpm/conf.d/20-newrelic.ini
+      - /etc/php/7.2/cli/conf.d/20-newrelic.ini
+      sansible_newrelic_license_key: 123456789123456789123456789123456789
```
Comment (wojtek-oledzki) [inline README.md]:
```suggestion
      sansible_newrelic_agent_php_license_key: 123456789123456789123456789123456789
```

File: README.md [hunk 5]
```diff
@@ -27,18 +28,58 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
...
 
   roles:
-    - role: sansible.newrelic_integration_php
+    - role: sansible.newrelic_agent_php
+      sansible_newrelic_agent_php_appname: some-php7.3-app
+      sansible_newrelic_agent_php_ini_config:
+        - option: newrelic.distributed_tracing_enabled
+          section: newrelic
+          value: "true"
+      sansible_newrelic_license_key: 123456789123456789123456789123456789
```
Comment (wojtek-oledzki) [inline README.md]:
```suggestion
      sansible_newrelic_agent_php_license_key: 123456789123456789123456789123456789
```

File: README.md [hunk 6]
```diff
@@ -27,18 +28,58 @@ and run `ansible-galaxy install -p ./roles -r roles.yml`
...
+
+```YAML
+- name: Install and configure newrelic
+  hosts: "somehost"
+
+  roles:
+    - role: sansible.newrelic
+      sansible_newrelic_agent_php_appname: some-php7.2-app
+      sansible_newrelic_agent_php_ini_paths_base:  /etc/php/7.2/mods-available/newrelic.ini
+      sansible_newrelic_agents_php_ini_paths_links:
```
Comment (wojtek-oledzki) [inline README.md]:
```suggestion
      sansible_newrelic_agent_php_ini_paths_links:
```

File: molecule/default/molecule.yml [hunk 7]
```diff
@@ -38,20 +27,14 @@ lint:
           max: 159
         truthy: disable
 
-# Dependency resolution is by default handled through Ansible Galaxy, but support for gilt (http://gilt.readthedocs.io/)
-# is available.  See https://molecule.readthedocs.io/en/latest/configuration.html#dependency for details.
 dependency:
   name: galaxy
+  options:
+    role-file: requirements.yml
```
Comment (wojtek-oledzki) [inline molecule/default/molecule.yml:33]: missing ignore certs?

```YAML
ignore-certs: True
```

File: tasks/configure.yml (hunk 2 above)
Comment (wojtek-oledzki) [reply tasks/configure.yml]:
```
{{ 'true' if item.value == true else 'false' if item.value == false else item.value }}
```

**Review by wojtek-oledzki — CHANGES_REQUESTED** [review]

File: molecule/default/molecule.yml (hunk 7 above)
Comment (lobsterdore) [reply molecule/default/molecule.yml:33]: Not needed now we are on Xenial.

File: tasks/configure.yml (hunk 2 above)
Comment (lobsterdore) [reply tasks/configure.yml:10]: That's way less readable, I prefer the original :)

**Review by wojtek-oledzki — APPROVED** [review]

**Pull request merged**